src/qt/guiutil.cpp

```diff
@@ -567,7 +567,10 @@ TableViewLastColumnResizingFixer::TableViewLastColumnResizingFixer(QTableView* t
 #ifdef WIN32
 boost::filesystem::path static StartupShortcutPath()
 {
-    return GetSpecialFolderPath(CSIDL_STARTUP) / "Bitcoin.lnk";
```

Comment:
Upstreaming it's the idea, see https://github.com/bitcoin/bitcoin/pull/6382 .
Assuming all those changes are merged upstream, the more of them we include, the easier it will be to rebase (specially if they're included in the base of the branch being rebased).